### ISSUE TITLE
Production volume - maker community 2 week target ?

### ISSUE THREAD

**Issue by danjstone** [Charlotte-MEDI/Faceshields#26]
Hi Guys, 

I wanted to get a discussion on our production volume forecast/demand. 

I assume we have aprox 1 week to 10 days before moving around will get locked down significalntly which will slow us down significantly. 

As we lock in on a design and start to plan and buy materials it is practically impossible to do so without some type of volume forecast/target. 

Lots of factors go into how long it will take us to manufacture (design/number of printers/materials/number of lasers/people hours etc). 

We can do the "best we can" but are likely to either over purchase or end up with the wrong mix of supplies. 

We have plastic for 10K face shields but current design time to 3D print we are unlikely to be able to  manufacture 10K in 2 weeks. 

Are we ready to set a production target for us as a maker community for the next 2 week? 

ds

**Comment by tjfagan** [comment]
I agree, I was just about to put this question out there. I am lining up manufactures who are willing and able to help, The volume amount would be a good piece of information. 

We have access to a company who can make 1000 face shields (transparent piece)/day out of PVC. Also, we need a medical professional interface so we can ask these types of questions. I will put this in another issue.

**Comment by thomasd538** [comment]
Also, I am getting several more makers this morning offering to help print.  I do agree we need to get this to industry as possible, we can ramp up to 50 -100 masks a day.   Not enough, but it helps until we can industry involved

**Comment by DavidCTaylorPhD** [comment]
Pros and Cons of casting a wider net for makers?

**Comment by JeremyProffitt** [comment]
Con: 
  More people making, the more chances of infecting medical staff?
  more quality control is required
  more attention needed for managing makers, dedicated person?
Pros:
  Higher quantities, the only way to hit 1000 in a week?
  More test cases to test different print materials/type

**Comment by stoph** [comment]
I agree we need more folks involved to hit those numbers, but we also have to balance peoples want to help and their ability. My concern is that anyone with a 3D printer would _want_ to help, but we could quickly get into a situation were our experts (i.e. Jeremy) are spending too much time helping people debug print issues. Even if it's just asking people to rate themselves on a scale of 1-5 and only take the 3+'s (for now)

Secondly, with more makers, logistics becomes an issues delivering and picking up prints. It might make more sense to have 10 makers who can produce 10 each rather than 100 who can produce 1 each.

What are some other areas they could help besides printing?

**Comment by ghost** [comment]
I'm a little biased (as a person with one hacker printer) but I'd strongly recommend keeping this open to the larger community. For functional reasons, it makes sense to have a community point person (not Jeremy!), although ideally, they have enough knowledge to help trouble-shoot.

There are a couple reasons (some cited above): 

1) There may come a point when, even with industrial production, community help is needed. (I can't stop thinking of all my WWII studies in this regard!). It's worth the trouble to nurture that part of the community now, since if we _do_ get to a point where demand exceeds industry supply, it will be much harder to coordinate later.  Right now, there's a lot of goodwill -- it's important to channel that somehow, even if that doesn't feed directly into designs used for industrial manufacture.  

2) If people come to this group wanting to help, and the group turns them away, it will foster some frustration and add to the overall sense of helplessness -- even hopelessness.  What's the single most important thing each of us can do right now? Wash our hands and stay home!  But we're social animals -- our human natures are at their best when we're most altruistic.  I know this may seem like a very minor concern, but I'm telling you right now -- those who serve the public *mental* health are also being stretched.  We need to help medical professionals holistically.

I'd recommend running the industry output numbers by the docs and state public health admin. 
 If they see the public help as more harm then good, well, there you go then.  But I think it's really worth assigning a point person (graduate students? advanced undergrads?) to help be the smaller community outreach and point person.    It may ultimately make more sense to actually split that group off entirely, but I think keeping a clear line of communication open is important. 

Anyway, just my thoughts on this, but I respect and appreciate whatever the rest of the group decides.

**Comment by stoph** [comment]
100% agree Heather. Sorry, I didn't mean to say we'd turn _anyone_ away, just want to make sure we have a process in place to handle the volunteers and someone dedicated to that effort. I just spoke with someone coordinating volunteers to help assemble too and there were a lot of questions I didn't have answers to **yet**.

**Comment by ghost** [comment]
Sorry, myself, hope I didn't sound crabby, just one of those home-school+online-class+printer-fails kind of days. :D 

I *do* think this would be a great thing for a grad student to take over (we don't have any in Art, but Alex might know some?)   

Tasks would/could include 
- Creating a PDF of specs for different machine types
- PDF (with images) of how to disinfect, bag, and label
- Pickup-delivery coordination with other volunteers without printers
- Organizing sub-groups of volunteers by type - those who can lasercut, those who can print, those who can run masks -- and if/when N95s run out, those who can sew surgical masks, gowns, etc. (That's another branch of inquiry, but no doubt relevant to the whole unless there's a tested and reliable N95 DIY design eventually.)
